static/js/panels/ProfileForm.tsx

```diff
@@ -75,6 +132,64 @@ const ProfileForm: FC = () => {
                 }
                 stacked
               />
+              <Accordion
+                titleElement="h3"
+                sections={[
+                  {
+                    content: (
+                      <>
+                        <NicDeviceSelector
+                          notify={notify}
+                          setNicDevice={setNicDevice}
+                        />
+                        <hr />
+                        <DiskDeviceSelector
+                          notify={notify}
+                          setDiskDevice={setDiskDevice}
+                        />
+                      </>
+                    ),
+                    title: "Devices",
```

Comment:
Wouldn't it make more sense to split them into two sections storage and network?

Reply:
It would make sense to do so if we allowed the user to add multiple networks and/or multiple storage volumes, but since now they are the only 2 options, I think grouping them into one "devices" section is better than having 2 sections with only 1 element inside each.